bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -538,19 +538,21 @@ uint32_t PreProcessor::launchReqPreProcessing(uint16_t clientId,
   // Unused for now. Replica Specific Info not currently supported in pre-execution.
   uint32_t replicaSpecificInfoLen = 0;
   auto span = concordUtils::startChildSpanFromContext(span_context, "bft_process_preprocess_msg");
-  requestsHandler_.execute(clientId,
-                           reqSeqNum,
-                           PRE_PROCESS_FLAG,
-                           reqLength,
-                           reqBuf,
-                           maxReplyMsgSize_,
-                           (char *)getPreProcessResultBuffer(clientId),
-                           resultLen,
-                           replicaSpecificInfoLen,
-                           span);
-  if (!resultLen)
-    throw std::runtime_error("Actual result length is 0 for clientId: " + to_string(clientId) +
-                             ", requestSeqNum: " + to_string(reqSeqNum));
+  auto error = requestsHandler_.execute(clientId,
```

Comment:
I would call the result variable just 'result'; 'error' is misleading.

Reply:
Thanks Yulia, makes sense! I decided to call it `status`, to avoid a confusion between `result` and `resultLen`.